gitpod-io/website
# add `get-a-demo` card under `/self-hosted` pricing

## Description

introduced a new `get-a-demo` card under `/self-hosted` pricing page

## Related Issue(s)



Fixes #2779

## How to test


- Go To [`/self-hosted`](https://deploy-preview-2972--gitpod-kumquat.netlify.app/self-hosted) & see :eyes:



<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2972"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>
